README.md

```diff
@@ -35,4 +77,33 @@
 - [ ] [Grafana](https://grafana.com) - metrics extension
 - [ ] [Prometheus](https://prometheus.io) - metrics extension
 - [ ] [Seq](https://datalust.co/seq) - logging extension
-- [ ] [Vault](https://www.vaultproject.io) - secrets extension
\ No newline at end of file
+- [ ] [Vault](https://www.vaultproject.io) - secrets extension
+
+
+## Code Coverage
+Stay informed about the code quality with our integrated Codecov badge that displays the current code coverage percentage:
+[![codecov](https://codecov.io/gh/SaintAngeLs/distributed_minispace/graph/badge.svg?token=SW3T9CN2QS)](https://codecov.io/gh/SaintAngeLs/distributed_minispace)
+
+## Contributing
+Contributions are what make the open-source community such an amazing place to learn, inspire, and create. Any contributions you make are **greatly appreciated**.
+
+Please refer to [CONTRIBUTING.md](./CONTRIBUTING.md) for more details.
+
+## License
+Distributed under the MIT License. See [LICENSE](./LICENSE) for more information.
```

Comment:
This links to license doesn't work, probably because of LICENCE file name.

Reply:
Now the link to licence should work properly.

Reply:
Yeah, that one in Contributing.md file works now, but we have one more at the bottom of Readme.md file.

Reply:
Very nice attention, thank you. Now it should link to the LICENCE file in the README.md correctly.